% XeLaTeX can use any Mac OS X font. See the setromanfont command below.
% Input to XeLaTeX is full Unicode, so Unicode characters can be typed directly into the source.

% The next lines tell TeXShop to typeset with xelatex, and to open and save the source with Unicode encoding.

%!TEX TS-program = xelatex
%!TEX encoding = UTF-8 Unicode

\documentclass[12pt]{article}

% See geometry.pdf to learn the layout options. There are lots.
\usepackage[papersize={8.5in,11in}, total={7in,9.5in},top=20mm, left=20mm, includefoot]{geometry}

\setlength{\parindent}{20pt}
\setlength{\parskip}{1pt}


\usepackage{graphicx}
\usepackage{amssymb}


%\usepackage{graphicx}[pdftex]
%\usepackage{wrapfig}
\usepackage{caption}
\linespread{0.95}
%\usepackage{setspace}
% Will Robertson's fontspec.sty can be used to simplify font choices.
% To experiment, open /Applications/Font Book to examine the fonts provided on Mac OS X,
% and change "Hoefler Text" to any of these choices.
\usepackage{xspace}
\usepackage{fontspec,xltxtra,xunicode}
\defaultfontfeatures{Mapping=tex-text}
\setromanfont[Mapping=tex-text]{Times New Roman}
\setsansfont[Scale=MatchLowercase,Mapping=tex-text]{Gill Sans}
\setmonofont[Scale=MatchLowercase]{Andale Mono}

%set the bibliography style
\usepackage[super,sort&compress,comma]{natbib}
%\bibliographystyle{natbib}
\bibliographystyle{unsrt}
\renewcommand\bibnumfmt[1]{\emph{\textbf{\small{#1)}}}}
\setlength\bibsep{2pt}

%set up a header
\usepackage{fancyhdr}
\pagestyle{fancy}
\lhead{GB Gloor} \chead{\textbf{CIHR CV}} \rhead{027428 }
\newcommand{\HRule}{\rule{\linewidth}{.8mm}}
\setlength{\headheight}{15pt}
\makeatletter
\renewcommand\section{\@startsection
	{section}{1}{0in}% %name level indent
	{0.1\baselineskip}%
	{0\baselineskip}%
	{\sffamily\bfseries\large}
	%
}
\makeatother

\makeatletter
\renewcommand\subsection{\@startsection
	{subsection}{2}{0.25in}% %name level indent
	{0.1\baselineskip}%
	{0\baselineskip}%
	{\textbf}%
}
\makeatother

\makeatletter
\renewcommand\subsubsection{\@startsection
	{subsubsection}{3}{0.25in}% %name level indent
	{0.1\baselineskip}%
	{0\baselineskip}%
	{\textit}%
}
\makeatother

\begin{document}
\section{Most significant recent contributions}~
I have made seminal contributions to fields as diverse as DNA repair, \emph{P} element repression, biocomputing and more recently molecular protein evolution and microbial community analyses. While originally trained as a geneticist and biochemist, I began working on molecular evolution in proteins and developed both theoretical and practical skills in computational biology. More recently, I have developed significant collaborations with a number of leading and emerging scientists, including Dr Gregor Reid an internationally known microbiologist and proponent of probiotics, Drs Emma Allen-Vercoe outstanding young researchers, and Dr. Johanne Allard. These collaborations have led to a number of highly cited papers, and exciting projects in progress. I have two sets of overlapping expertise:\\

First, I have an interest in protein molecular coevolution that began when a group in Biochemistry and Applied Mathematics at Western coalesced to develop methods to identify and characterize co-evolving positions in protein families. Our goal was to find interacting sites within and between proteins. This work has led to an NSERC grant and the papers have together over 400 citations and multiple \emph{Faculty of 1000} recommendations. The Molecular Biology and Evolution paper published in 2010 received this review in its entirety:`This is an important paper. It deserves to be published'. I am continuing this work with a graduate student who has discovered that covarying sites in close proximity represent poorly aligned regions. Finally, as part of this work, I have begun collaborating with Dr David Edgell on molecular evolution of homing endonucleases. This has led to 4 papers so far, two of which were published in PNAS.\\

Second, my group has developed significant expertise in a number of areas relating to high throughput sequencing:
\begin{enumerate} \setlength{\itemsep}{-4pt} 
\item I developed a novel method to characterize microbiota samples using the Illumina sequencing technology. This has led to 17 papers so far, an invitation to speak to the Ontario Illumina Users Group, the U. Maryland Genome Sciences Centre, and being a co-applicant on the successful Vaginal Microbiota Group Grant, two grants with Dr. Allard and one with David Spence. I collaborated with Dr Allen-Vercoe and Dr Elaine Petrof to characterize the microbiota of {\em Clostridium difficile} patients who  received a synthetic colon microbiota --- resulting in a cure for the disease. The resulting paper in {\em Microbiome} among the top 1\% most cited papers at BioMed Central, and I was co-first author.
\item Our group has developed expertise in {\em de-novo} genome sequencing (Macklaim et al 2011 PNAS, Anukam et al 2013, PLoS ONE).
\item We developed a novel statistical method for meta-RNA-seq that uses Baysian techniques to conduct a consistency check on gene expression levels. With this, we can identify gene expression changes that are not linked to organism abundance (Fernandes, PLoS ONE 2013). We have generalized the method to work with any high throughput sequencing dataset including RNA-seq, 16S rRNA gene sequencing and differential growth assays (Fernandes et al, Microbiome 2014). This methodology, for the first time, allows us to interrogate mixed microbial communities and find out what they are doing, not just who is there in a manner that is not susceptible to false positive identification because of limited sample sizes.
\end{enumerate}
\newpage
\section{Research Contributions and Practical Applications}
\subsection*{Recent Invited presentations:} 
\begin{itemize}    
\setlength{\itemsep}{0pt}%
    \setlength{\parskip}{0pt}%
%\item{Invited speaker at University of North Carolina-Charlotte Department of Bioinformatics, 2008} 
\item{Invited platform speaker at the Ontario Illumina Users Group 2010}
\item{Invited  speaker at the RePOOPulating the gut: therapeutic microbial preparations to eradicate recurrent {\em C.difficile} infections in Canada, Toronto Oct 2011}
\item{Invited expert participant at International Society for the Application of Probiotics and Prebiotics, Berkley, CA Oct 2011}
\item{Invited speaker at Fondation Merieux Conference on Better Foods for Better Health, Annecy, France, Sept 2013}
\item {Invited expert participant at African International Conference and Workshop on the Microbiome and Probiotics, Nairobi, Kenya, Nov 2013}
\item{Invited speaker at the Institute of Genome Sciences seminar series, University of Maryland, Balti- more, USA Oct 2013}

\end{itemize}
%}

%\renewcommand{\tab}{\hspace{1.5in}}


%\renewcommand{\tab}{\hspace{0.25in}}
\renewcommand\refname{}

\subsection*{Peer-reviewed papers published since 2010:}\cite{kernohan:2014, mcmurrough:2014,fernandes:2014,Urbaniak:2014aa,Rahat-Rozenbloom:2014aa,Gan:2014aa,rosenthal:2014,Dickson:2014aa,Di-Bella:2013aa,DaSilva:2013aa,Lahiry:2013aa,fernandes:2013,Burton:2013,macklaim:2013,Kim:2013aa,Petrof:2013aa,MacPhee:2013aa,Allen-Vercoe:2012,Anukam:2013aa,Allen-Vercoe:2012a,Macklaim:2012,Genereaux:2012,Kvas:2012,Dickson:2012,Turowec:2011a,Takeuchi:2011a,Macklaim:2011,Hummelen:2011,Reid:2011a,Duncan:2011,Hoke:2010a,Dickson:2010,Gloor:2010a,Fernandes:2010b,Kleinstiver:2010,Duncan:2010,Fernandes:2010a,Hummelen:2010,Gloor:2010}
\vspace{-5mm}
\bibliography{bibdesk_refs}
~\\
\textbf{Non-peer reviewed manuscripts submitted or under revision:}

Russell J Dickson and Gregory B Gloor. Xorro: Rapid paired-end read overlapper. arXiv preprint arXiv:1304.4620, 2013.\\

Russell J Dickson and Gregory B Gloor.The MIp  toolset:an efficient algorithm for calculating mutual information in protein alignments. arXiv preprint arXiv:1304.4573, 2013.
~\\
\textbf{Software:}\\
ALDEx2. ALDEx tool to examine compositional high-throughput sequence data with Welchâ€™s t-test and Wilcoxon rank test. https://github.com/ggloor/ALDEx2, last update May 14, 2014

\subsection*{Conference Proceedings and Abstracts:}~
$>$10 abstracts since 2010, all subsequently noted as published or 
submitted above


%\newpage
\section{Other Evidence of Impact and Contributions} 
\subsection*{Peer Review Committee memberships:}~

\begin{itemize}
    \setlength{\itemsep}{0pt}%
    \setlength{\parskip}{0pt}%

\item{Editorial Board: Microbiome (www.microbiomejournal.com)}
\item{Member CIHR Genetics panel 2010-present}
\item{IODE Doctoral Scholarship committee, 2008 - present}
\item{NCIC Model Organisms Panel B2, 2006-2010}
\item{MRC/CIHR BMB/Genetics/Genomics invitee, 1998, 1999, 2000, 2003, 2004, 2005, 2006, 2007, 2008}
\item{Peer review organizer for the Foundation for Gene and Cell Therapy, 1995-2001}
\item{OGS Biochemistry/Biophysics panel 1997-1999 (\textbf{Chair 1999})}
\item{NCIC Virology and Molecular Biology Committee, 1997 -- 2000}
\end{itemize}

\subsection*{External Evaluation of Grant Applications and Manuscripts:}~Regular 
external peer reviewer for MRC/CIHR since 1993 (average 2-3/year). 
Occasional external reviewer for NSERC, NSF, Genome Canada, and NCIC (average about 
1/year). Regular external reviewer for \emph{Genetics, Bioinformatics, PLOS ONE, and PLOS Computational Biology} (average 
about 2/year each). Occasional reviewer for \emph{PNAS, Mol. Cell. Biol., 
Genome, Trends in Genetics, Developmental Biology, Developmental 
Genetics,} etc. (about 2/year of this group).


%\newpage
\section{Delays in Research Activity}
~\\
None
\section{Contributions to the Training of Highly Qualified Personnel}

\subsection*{Graduate Student Training and Development:}~Member of Departmental 
Graduate Studies Committee, 1993-1996, 2003-2007 (\textbf{Chair, 
2004 -- 2007}). Have graduated 2 M.Sc. students, 3 Ph.D. students, 
and currently supervise 2 Ph.D. students and 1 MScs student. Served as University 
Examiner on over 20 thesis defences, and been the invited external 
Ph.D. examiner five times (one international).

\subsection*{Undergraduate Student Training and Development:}~Member of Departmental 
Undergraduate Studies Committee, 2009-present. Have supervised one honours student thesis topic annually since 1993. Winner of multiple undergraduate teaching awards:
\begin{itemize}
    \setlength{\itemsep}{0pt}%
    \setlength{\parskip}{0pt}%

\item 2004 - W.L. Magee award for excellence in teaching undergraduate biochemistry
\item 2006 - Schulich Excellence Award for Undergraduate Education
\item 2007 - University Student's Council Teaching Honour Roll
\item 2009 - University Student's Council Teaching Honour Roll
\item 2011 - University Student's Council Teaching Honour Roll
\item 2012 - University Student's Council Teaching Honour Roll
\end{itemize}

%\subsection*{Participation in Scientific Outreach:}~Designed and implemented 
%a teacher training program in biotechnology in collaboration 
%with Dr. Chris Brandl (2001 -- present). Teachers trained in this 
%program are provided access to the ``Biotech lab in a box'' made 
%with the aid of an NSERC grant for outreach. Designed and implemented 
%an outreach program for area OAC biology students (in partnership 
%with Dr. Chris Brandl). Personally instructed an average of 5 
%OAC biology classes per term (150 students/term), for three classes 
%each.

%\section*{Significant University Administrative Activities (Past 7 Years)}
%\subsection*{University Committees}
%Jul 2007 - Present Jul 2007 - Present Sep 2005 - Jul 2006
%6 files, 1 appeal Sep 2004 - Dec 2006
%Sep 2003 - Dec 2004 Sep 2001 - Dec 2002 Jul 2001 - Jun 2003 Jul 2000 - Jun 2003 Jul 19Contributions to the Training of Highly Qualified Personnel97 - Jun 2003
%Faculty Committees
%Mar 2008 - Mar 2008 Nov 2007 - Jul 2008 Feb 2007 - Present
%Member, Biology P\&T committee Member, Computer Science P\&T committee Member, Promotion and Tenure - Ivey School of Business
%Member, Promotion and Tenure - Pharmacology and Physiology Department Member, Promotion and Tenure - Biology Department Member, Promotion and Tenure - Biology Department Member, UWO Bioinformatics Search Committee
%Chair, Senate Subcommittee on the WWW Member, Senate Subcommittee on Computing and Information Technology
%Member, Margaret Moffat Day Member, Epidemiology and Biostatistics Chair Selection Committee Member, Ad hoc disciplinary committee
%Disciplinary committee to deal with case of teaching misconduct Dec 2005 - Jul 2007	Chair, ISSTF-Accreditation
%Faculty Accreditation Committee Jan 2001 - Jan 2003	Member, Human Molecular Genetics Selection Committee
%Faculty of Medicine
%Departmental Committees
%Jul 2004 - Jun 2008 Jul 2002 - Jun 2004 Jul 2001 - Jun 2004
%Biochemistry Jan 1998 - Jun 2004
%Biochemistry Jan 1995 - Jun 2005
%Biochemistry
%f)	Graduate Supervision
%Doctoral Thesis Masters Thesis Post-Doc Fellows Masters Committees Doctoral Committees
%Chair, Biochemistry Graduate Studies Member, Biochemistry Graduate Studies Chair, Visiting Speakers Committee
%Co-Chair, Department Outreach Committee Member, Ad hoc Computing Committee
%
\end{document}
