---
title: "CV: Gloor GB, PhD"
geometry: margin=2.5cm
urlcolor: blue
classoption: onecolumn
output:
    pdf_document:
        keep_tex: false
        fig_caption: false
        toc: false
        toc_depth: 1
    pandoc_args: [
            "-V",
        ]
---

## Current Position:

Professor and Chair, Department of Biochemistry, University of Western Ontario

## Relevant Expertise:

1) Protein evolution. I use and develop tools to examine how protein structure and function is maintained in response to sequence changes. I have a special interest in identifying the role that variable positions play in protein evolution. I have long taught senior undergraduate courses in protein sequence alignment and annotation.

2) Composition and function of the human and other microbiomes. I use and develop tools to examine 16S rRNA gene composition, gene content and expression of mixed population samples (meta-genomes and meta-transcriptomes), and metabolomic analysis of clinical samples. I have experience in metagenomic binning, sequence annotation and the analysis of shotgun DNA sequence data.

3) Computational biology and the application of techniques for compositional data analysis to the above problems. My primary contribution is ALDEx2 tool in Bioconductor for the analysis of high throughput experiments that generate counts per sequence tag: 16S rRNA gene sequencing, transcriptomics and selex-type experiments. I teach graduate courses, and have delivered many workshops, on the use of compositional data analysis techniques to examine transcriptomes, microbiomes and other types of complex data sets derived from high throughput sequencing.

## Software packages

- ALDEx2 - top 20% on Bioconductor: https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html

- CoDaSeq - public tool for compositonal data munging and display: https://github.com/ggloor/CoDaSeq

- omicplotR -  Shiny app to explore compositioanal data: https://www.bioconductor.org/packages/release/bioc/html/omicplotR.html

## Selected papers since 2015 (of 52)

### Methods papers

- D. J. Giguere, A. T. Bahcheli AT, B. R. Joris, J. M. Paulssen, L. M. Gieg, M. W. Flatley, G. B. Gloor (2020) Complete and validated genomes from a metagenome biorXIV: 10.1101/2020.04.08.032540

- D. J. Giguere, J. M. Macklaim, B. Y. Lieng, and G. B. Gloor, “omicplotr: visualizing omic datasets as compositions,” BMC Bioinformatics, vol. 20, p. 580, Nov 2019.

-  T. P. Quinn, I. Erb, G. Gloor, C. Notredame, M. F. Richardson, and T. M. Crowley, “A field guide for the compositional analysis of any-omics data,” Gigascience, vol. 8, Sep 2019.

- J. M. Macklaim and G. B. Gloor, “From RNA-seq to biological inference: Using compositional data analysis in meta-transcriptomics,” Methods Mol Biol, vol. 1849, pp. 193–213, 2018.

- J. J. Egozcue, V. Pawlowsky-Glahn, and G. B. Gloor, “Linear association in compositional data anal- ysis,” Austrian Journal of Statistics, vol. 47, pp. 3–31, january 2018.

- G.B. Gloor,J.M. Macklaim,V. Pawlowsky-Glahn,and J.J. Egozcue,“Microbiome datasets are compositional: And this is not optional,” Front Microbiol, vol. 8, p. 2224, 2017.

- G.B. Gloor,J.M. Macklaim,M. Vu,and A.D. Fernandes,“Compositional uncertainty should not be ignored in high-throughput sequencing data analysis,” Austrian Journal of Statistics, vol. 45, pp. 73– 87, September 2016.

- G. B. Gloor and G. Reid, “Compositional analysis: a valid approach to analyze microbiome high- throughput sequencing data,” Can J Microbiol, vol. 62, pp. 692–703, Aug 2016.

- G.B. Gloor, J.R. Wu,V. Pawlowsky-Glahn, and J.J. Egozcue,“It’s all relative: analyzing microbiome data as compositions,” Ann Epidemiol, vol. 26, pp. 322–9, May 2016.


- G. B. Gloor, J. M. Macklaim, and A. D. Fernandes, “Displaying variation in large datasets: Plotting a visual summary of effect sizes,” Journal of Computational and Graphical Statistics, vol. 25, no. 3C, pp. 971–979, 2016.

- R. G. Wong, J. R. Wu, and G. B. Gloor, “Expanding the UniFrac toolbox,” PLoS One, vol. 11, no. 9, p. e0161196, 2016.

-  A. McMillan, J. B. Renaud, G. B. Gloor, G. Reid, and M. W. Sumarah, “Post-acquisition filtering of salt cluster artefacts for LC-MS based human metabolomic studies,” J Cheminform, vol. 8, no. 1, p. 44, 2016.

### Genomics

-   A. Almeida, A. L. Mitchell, M. Boland, S. C. Forster, G. B. Gloor, A. Tarkowska, T. D. Lawley, and R. D. Finn, “A new genomic blueprint of the human gut microbiota,” Nature, vol. 568, p. 499, Feb 2019.

- M. D. Berg, D. J. Giguere, J. S. Dron, J. T. Lant, J. Genereaux, C. Liao, J. Wang, J. F. Robinson, G. B. Gloor, R. A. Hegele, P. O’Donoghue, and C. J. Brandl, “Targeted sequencing reveals expanded ge- netic diversity of human transfer rnas,” RNA Biol, vol. 16, pp. 1574–1585, Aug 2019.

- M. I. Petrova, J. M. Macklaim, S. Wuyts, T. Verhoeven, J. Vanderleyden, G. B. Gloor, S. Lebeer, and G. Reid, “Comparative genomic and phenotypic analysis of the vaginal probiotic lactobacillus rham- nosus GR-1,” Front Microbiol, vol. 9, p. 1278, 2018.

### Protein evolution and engineering

- T.A. Hamilton, G.M. Pellegrino, J.A. Therrien, D.T. Ham, P.C. Bartlett, B.J. Karas, G.B. Gloor, and D. R. Edgell, “Efficient inter-species conjugative transfer of a crispr nuclease for targeted bacterial killing,” Nat Commun, vol. 10, p. 4544, Oct 2019.

- M. Laforet, T. A. McMurrough, M. Vu, C. M. Brown, K. Zhang, M. S. Junop, G. B. Gloor, and D. R. Edgell, “Modifying a covarying protein-dna interaction changes substrate preference of a site- specific endonuclease,” Nucleic Acids Res, vol. 47, pp. 10830–10841, Nov 2019.

- M. D. Berg, J. Genereaux, Y. Zhu, S. Mian, G. B. Gloor, and C. J. Brandl, “Acceptor stem differences contribute to species-specific use of yeast and human tRNAser,” Genes (Basel), vol. 9, p. 612, Dec 2018.

- T. A. McMurrough, C. M. Brown, K. Zhang, G. Hausner, M. S. Junop, G. B. Gloor, and D. R. Edgell, “Active site residue identity regulates cleavage preference of LAGLIDADG homing endonucleases,” Nucleic Acids Res, vol. 46, pp. 11990–12007, Oct 2018.

## National/International Conference Speaking and Workshops (past 5 years)

- 2019, Invited workshop speaker, Computational and Methodological Statistics 2019, London, England

- 2019, Invited Speaker, Emerging Challenges in Microbiome Data Analysis, BIRS, Banff, Canada

- 2019, Invited Platform Speaker, Barcelona Microbiome Debates, Barcelona, Spain

- 2019, Invited Webinar Presenter, International Society for the Application of Probiotics and Probiotics (ISAPP)

- 2019, Invited Speaker, North American Microbiome Congress, Washington, USA 2018,

- 2019, Invited Webinar Presenter, International Life Sciences Institute (ILSI)

- 2018, Invited industrial speaker (KGK Biosciences), Supply Side West, Las Vegas, USA

- 2018, Invited speaker, EMBL-EBI, EBI Seminar Series, Hinxton, Cambridgeshire, UK

- 2018, Workshop organizer and presenter, Compositional Data analysis methods, NGS’18, Barcelona, Catalonia

- 2017, Keynote, Microbial Ecology 2017, Toronto, Ontario

- 2017, Invited speaker, EMBL-EBI Industrial Program Workshop - The human microbiome: challenges and opportunities for novel therapeutics, Hinxton, England

- 2017, Canadian Statistical Sciences Institute Microbiome Planning Meeting speaker and discussion leader, Winnipeg,Manitoba

- 2016, Invited speaker at Exploring Human Host-Microbiome Interactions in Health and Disease 2016, Cambridge, UK

- 2016, Invited workshop organizer at Exploring Human Host-Microbiome Interactions in Health and Dis- ease 2016, Cambridge, UK

- 2016, Invited workshop presenter, The Human Microbiome and Epidemiology, 2016 Epidemiology Congress of the Americas, Miami, USA

- 2016, Invited workshop, Infection, Inflammation and Immunity course, The Arctic University of Norway, Tromso, NO





